test/unit/api/compress/compress_copy_test.hpp

```diff
@@ -151,7 +174,8 @@ compress_copy_tst_mask(eve::as<T> tgt, L m, auto algo)
       for( int j = 0; j != T::size() - i; ++j ) { for_ignore_tests(eve::ignore_extrema(i, j)); }
     }
   }
-  else if constexpr( eve::has_bundle_abi_v<T> && eve::current_api >= eve::sve )
+  else if constexpr( eve::has_bundle_abi_v<T> && eve::current_api >= eve::sve
+                     && eve::current_api >= eve::rvv )
```

Comment:
&& doesn't seem right. this is always false.

Reply:
Yep, fixed.